sysinit/coqargs.ml

```diff
@@ -294,7 +294,10 @@ let parse_args ~usage ~init arglist : t * string list =
       }}
 
     |"-compat" ->
-      let xy = String.(concat "" ("Coq" :: split_on_char '.' (next ()))) in
+      let arg = next () in
+      let rocq_name = if arg <> "" && arg.[0] = '8' then "Coq" else "Rocq" in
+      (* remove the above and replace by "Rocq" once theories/Compat/Coq820.v is removed *)
+      let xy = String.(concat "" (rocq_name :: split_on_char '.' arg)) in
```

Comment:
To be future-proof for Rocq 80.0, we should probably check the full first version component and not only the first character. Since we are already splitting on `.` later in the same function, what about doing it earlier and checking the head of the resulting list?

Reply:
I hope we will get rid of this temporary code before Rocq 80, but indeed why not.